### PR TITLE
Add influxdb-rails

### PR DESCRIPTION
**Pull request by ChrisBr**
to collect performance metrics of the OBS rails app like
* request time
* view render time
* db query time

This information will be collection on metrics.o.o

The influxdb is already reachable on metrics.o.o so we could deploy it. Let me know so I can give you the credentials.

### CODE REVIEW

File: src/api/config/options.yml.example [hunk 1]
```diff
@@ -183,6 +183,14 @@ default: &default
   #    persistent: true
   #    passive: true
 
+  # For storing rails metrics in influxdb
```
Comment (hennevogel) [inline src/api/config/options.yml.example]: This file is for Admins. I would suggest:

`# For sending application performance metrics to a influx time series database`

File: src/api/config/options.yml.example [hunk 2]
```diff
@@ -183,6 +183,14 @@ default: &default
...
+  # https://github.com/influxdata/influxdb-rails
```
Comment (hennevogel) [inline src/api/config/options.yml.example]: For admins this url doesn't really have much value as it does not explain any of the settings.

File: src/api/config/options.yml.example [hunk 3]
```diff
@@ -183,6 +183,14 @@ default: &default
...
+  # influxdb_database: rails
+  # influxdb_username:
+  # influxdb_password:
+  # influxdb_hosts:
```
Comment (hennevogel) [inline src/api/config/options.yml.example]: Please fill those with example data, so people at least have the chance to know what is expected.

**Comment by ChrisBr** [comment]
> +1 the only thing I'm worried about is config.retry...

Why that? So you want to disable the retry or increase or decrease or...?

**Comment by hennevogel** [comment]
See [Q: How does this gem handle an unreachable InfluxDB server?](https://github.com/influxdata/influxdb-rails#frequently-asked-questions) in the gems FAQ.

**Comment by ChrisBr** [comment]
> See Q: How does this gem handle an unreachable InfluxDB server? in the gems FAQ.

Yup, I know, that's why I limited to 10 retries. I think it is not critical when we loose some of the data points if InfluxDB is not available.

**Pull request merged**